sound/soc/sof/intel/Kconfig

```diff
@@ -330,6 +331,7 @@ config SND_SOC_SOF_HDA_AUDIO_CODEC
 
 config SND_SOC_SOF_HDA_ALWAYS_ENABLE_DMI_L1
 	bool "SOF enable DMI Link L1"
+	depends on SND_SOC_INTEL_UNSUPPORTED || COMPILE_TEST
```

Comment:
Hmm, this popped up a bit as a different kind of unsupport option, but I guess it still fits the bill.